source/diagnostics.tex

```diff
@@ -1192,7 +1201,6 @@
 
   // \ref{syserr.errcode.nonmembers} non-member functions:
   error_code make_error_code(errc e) noexcept;
-  bool operator<(const error_code& lhs, const error_code& rhs) noexcept;
 
   template <class charT, class traits>
     basic_ostream<charT, traits>&
```

Comment:
I feel that everything should be in the header synopsis, otherwise that synopsis is incomplete. Whether some non-members are also duplicated in the class synopses is less important, but I made the call based on whether they are defined in a subclause of the class, or in a sibling clause.